tests/main/base-policy/task.yaml

```diff
@@ -19,6 +19,12 @@ prepare: |
     "$TESTSTOOLS"/snaps-state install-local test-snapd-sh-core
     "$TESTSTOOLS"/snaps-state install-local test-snapd-sh-core18
 
+    # core26 is currently only available on edge
+    if ! snap list core26 2> /dev/null; then
+        snap install --edge core26
```

Comment:
@sergiocazzolato do we need to manually remove core26 or are bases cleaned up as well?

Reply:
We remove in the cleanup the bases as well, so it is not needed to remove then in the tests